jobclass/rebuild-rename.rb

```diff
@@ -21,18 +21,18 @@
 
   script {|params, script|
     script.task(params['data-source']) {|task|
-      dest_table = '$dest_table'
-      prev_table = '${dest_table}_old'
-      work_table = '${dest_table}_wk'
+      dest_table = params['dest-table']
+      prev_table = TableSpec.parse("#{dest_table}_old")
+      work_table = TableSpec.parse("#{dest_table}_wk")
```

Comment:
👈  [check] prev_table and work_table has only table name without schema name.
https://github.com/bricolages/bricolage/blob/a53284f0db8f795fb1a51aff8729cd53faccda5b/lib/bricolage/sqlstatement.rb#L304-L326